### ISSUE TITLE
Lua error purchasing Warband bank slot

### ISSUE THREAD

**Issue by Licen-it** [Jaliborc/Bagnon#1930]
**Which software were you running?**
 - Addon version name: v11.0.5
 - Client used: [e.g. Vanilla 1.14.4, Retail PTR x.x.x, etc] Retail

**Have you read the changelog?**
 Mark if read.
Yes

**Please describe the bug.**
Purchasing a new warband bank slot is throwing the lua error linked below

**Please describe how to reproduce it.**
1. Go to the bank
2. Click on unpurchased warband bank slot and buy it
3. See error

**Error Logs are Important!**
Date: 2024-08-22 00:04:29
ID: 1
Error occured in: AddOn: Bagnon
Count: 1
Message: Error: AddOn Bagnon attempted to call a forbidden function (PurchaseBankTab()) from a tainted execution path.
Debug:
   [string "=[C]"]: PurchaseBankTab()
   [string "@Interface/AddOns/BagBrother/frames/containers/bankTab.lua"]:33: method()
   [string "@Interface/AddOns/BagBrother/libs/Sushi-3.2/classes/core/Callable.lua"]:59:
      .../BagBrother/libs/Sushi-3.2/classes/core/Callable.lua:55
   [string "@Interface/AddOns/BagBrother/libs/Sushi-3.2/Sushi-3.2.lua"]:28: FireCalls()
   [string "@Interface/AddOns/BagBrother/libs/Sushi-3.2/classes/groups/Popup.lua"]:199: Accept()
   [string "@Interface/AddOns/BagBrother/libs/Sushi-3.2/classes/groups/Popup.lua"]:182: method()
   [string "@Interface/AddOns/BagBrother/libs/Sushi-3.2/classes/core/Callable.lua"]:59:
      .../BagBrother/libs/Sushi-3.2/classes/core/Callable.lua:55
   [string "@Interface/AddOns/BagBrother/libs/Sushi-3.2/Sushi-3.2.lua"]:28: FireCalls()
   [string "@Interface/AddOns/BagBrother/libs/Sushi-3.2/classes/buttons/Clickable.lua"]:38:
      ...Brother/libs/Sushi-3.2/classes/buttons/Clickable.lua:34
Locals:

**Comment by Protonull** [comment]
Yup, also experiencing this on v11.0.20 on retail. Curiously not getting any errors on `/console scriptErrors 1` though, but it's definitely Bagnon: [BagnonNotWorking.webm](https://github.com/user-attachments/assets/a9becc14-867f-4b8f-b468-396901652d4a)
